tools/test-framework/src/chain/ext/transfer.rs

```diff
@@ -129,18 +176,45 @@ impl<'a, Chain: Send> ChainTransferMethodsExt<Chain> for MonoTagged<Chain, &'a C
         num_msgs: usize,
         memo: Option<String>,
     ) -> Result<(), Error> {
-        let rpc_client = self.rpc_client()?;
-        self.value().runtime.block_on(batched_ibc_token_transfer(
-            rpc_client.as_ref(),
-            &self.tx_config(),
-            port_id,
-            channel_id,
-            sender,
-            recipient,
-            token,
-            num_msgs,
-            memo,
-        ))
+        match self.value().chain_type {
+            ChainType::Axon => {
+                let websocket_addr = self.value().websocket_address();
+                let home_path = self.value().home_path.clone();
+                self.value().runtime.block_on(async {
+                    for _ in 0..num_msgs {
+                        crate::relayer::axon::transfer::ibc_token_transfer(
+                            websocket_addr.clone(),
+                            home_path.clone(),
+                            port_id,
+                            channel_id,
+                            sender,
+                            recipient,
+                            token,
+                            None,
```

Comment:
Is it possible to use `timeout` as the arg now?

Reply:
Which `timeout`, the interface of this function do not has one.

Reply:
ops, just see https://github.com/synapseweb3/forcerelay/pull/330#discussion_r1334108648